packages/tpetra/core/src/Tpetra_CrsGraph_decl.hpp

```diff
@@ -2809,7 +2811,7 @@ namespace Tpetra {
       typedef CrsGraph<LocalOrdinal, GlobalOrdinal, InputNodeType> input_crs_graph_type;
       typedef CrsGraph<LocalOrdinal, GlobalOrdinal, OutputNodeType> output_crs_graph_type;
 
-      static Teuchos::RCP<output_crs_graph_type>
```

Comment:
We don't necessarily have to deprecate this, since it lives in the Details namespace and is thus fair game for instant removal, though it doesn't hurt.